### PR TITLE
#71 | Event gallery

### PR DESCRIPTION
**Pull request by Omarkojak**

### CODE REVIEW

File: app/Event.php [hunk 1]
```diff
@@ -85,4 +85,9 @@ public function scopeBetweenTiming($query, $start, $end)
     {
         $query->whereBetween('timing', [$start, $end]);
     }
+
+    public function images()
+    {
```
Comment (AhmadElsagheer) [inline app/Event.php:90]: photos

File: database/migrations/2016_04_05_162045_create_images_table.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('images', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('event_id');
```
Comment (AhmadElsagheer) [inline database/migrations/2016_04_05_162045_create_images_table.php]: foreign key referencing is missing

File: database/migrations/2016_04_05_162045_create_images_table.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateImagesTable extends Migration
+{
```
Comment (AhmadElsagheer) [inline database/migrations/2016_04_05_162045_create_images_table.php]: CreatePhotosTable

File: database/migrations/2016_04_05_162045_create_images_table.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('images', function (Blueprint $table) {
```
Comment (AhmadElsagheer) [inline database/migrations/2016_04_05_162045_create_images_table.php]: photos

File: database/migrations/2016_04_05_162045_create_images_table.php [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::drop('images');
```
Comment (AhmadElsagheer) [inline database/migrations/2016_04_05_162045_create_images_table.php]: photos

**Comment by AhmadElsagheer** [comment]
Rename your branch to follow the convention
http://blog.changecong.com/2012/10/rename-a-remote-branch-on-github/

**Comment by Omarkojak** [comment]
hwa eh elconvention b2a m3lsh :D

**Comment by AhmadElsagheer** [comment]
lowercase letters

**Pull request closed without merging**